lib/index.js

```diff
@@ -99,7 +99,15 @@ function nodeNetCatSrc(port, input) {
 }
 
 const FUNCTION_PRIORITY = [nativeNC, nodeNC];
-const FLAGS = isElectron() ? ['--ms-enable-electron-run-as-node'] : [];
+const needElectronFlag = vsCodeVersion => {
+  if (vsCodeVersion == null) {
+    return false;
+  }
+  const [major, minor] = vsCodeVersion.split('.');
+  // from 1.86, electron is enabled by default.
+  return +major <= 1 && +minor <= 85;
+}
+const FLAGS = isElectron() && needElectronFlag(process.env.$vscode_version || _VSCODE_PACKAGE_JSON?.version) ? ['--ms-enable-electron-run-as-node'] : [];
```

Comment:
why don't we just pass this flag into the api call?

Reply:
We have multiple places that needs this FLAG. Not sure I understand correctly on your previous comment, but we need to do parsing on the `version` value and compare it with 1.86. If it is equal or later, we don't need the flag, otherwise, we have to use it.

Reply:
This is sync-rpc, it should have no references to VsCode. That needs to be handled from our side in c3vsce repo and it sets process.env.$electronFlags or something which would be used here. We cannot update the file every time vscode decides to add a new flag requirement.